src/services/cart.js

```diff
@@ -0,0 +1,111 @@
+const  { Cart, CartItem  }= require('../config');
+
+class CartService{
+    async createCart (userId){
+        //Create the cart adding the User id as FK
+        console.log('Id', userId);
+        const cart =Cart.build({ 
+            status: 'new',
+            total: 0,
+            date: new Date(),
+            UserId: userId
+        });
+        await cart.save();
+        return cart
+    }
+    
+    async findAllCarts(){
+        const carts = await Cart.findAll();
+        if (carts===null){
+            throw new Error('There are no existing Carts');
+        }else{
+            return carts;
+        }
+    }
```

Comment:
If we are not using this method and would not be required in the future, I think we should remove this one

Reply:
done and removed!